decidim-assemblies/lib/decidim/assemblies/menu.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require "decidim/seeds"
+
+module Decidim
+  module Assemblies
+    class Menu
+      def self.register_engine_menu!
+        Decidim.menu :menu do |menu|
+          menu.add_item :assemblies,
+                        I18n.t("menu.assemblies", scope: "decidim"),
+                        decidim_assemblies.assemblies_path,
+                        position: 2.2,
+                        if: OrganizationPublishedAssemblies.new(current_organization, current_user).any?,
+                        active: :inclusive
+        end
+
+        Decidim.menu :home_content_block_menu do |menu|
```

Comment:
this could be extracted in its own method.

Reply:
Done in 508f641265